rclcpp/include/rclcpp/node_interfaces/node_parameters.hpp

```diff
@@ -89,6 +89,13 @@ class NodeParameters : public NodeParametersInterface
     const std::string & name,
     rclcpp::Parameter & parameter) const;
 
+  RCLCPP_PUBLIC
+  virtual
```

Comment:
I think adding a virtual method breaks ABI

Reply:
Never mind, I found another way.  Update to this PR coming.